modules/installation-vsphere-config-yaml.adoc

```diff
@@ -13,35 +13,53 @@ parameters.
 ----
 apiVersion: v1
 baseDomain: example.com <1>
+compute:
+- hyperthreading: Enabled <2>
+  name: worker
+  replicas: 0 <3>
 controlPlane:
+  hyperthreading: Enabled <2>
   name: master
-  replicas: 3 <2>
+  replicas: 3 <4>
 metadata:
-  name: test <3>
+  name: test <5>
 platform:
   vsphere:
-    vcenter: your.vcenter.server <4>
-    username: username <5>
-    password: password <6>
-    datacenter: datacenter <7>
-    defaultDatastore: datastore <8>
-pullSecret: '{"auths": ...}' <9>
-sshKey: 'ssh-ed25519 AAAA...' <10>
+    vcenter: your.vcenter.server <6>
+    username: username <7>
+    password: password <8>
+    datacenter: datacenter <9>
+    defaultDatastore: datastore <10>
+pullSecret: '{"auths": ...}' <11>
+sshKey: 'ssh-ed25519 AAAA...' <12>
 
 ----
 <1> The base domain of the cluster. All DNS records must be sub-domains of this
 base and include the cluster name.
-<2> The number of control plane machines that you add to the cluster. Because
+<2> Whether to enable or disable simultaneous multithreading, or
+`hyperthreading`. By default, simultaneous multithreading is enabled
+to increase the performance of your machines' cores. You can disable it by
+setting the parameter value to `Disabled`. If you disable simultanous
+multithreading in some cluster machines, you must disable it in all cluster
+machines.
++
+[IMPORTANT]
+====
+If you disable simultaneous multithreading, ensure that your capacity planning
+accounts for the dramatically decreased machine performance.
```

Comment:
For HT disabled scenarion there would be a memory and CPU requirement of at least

num_cpus: "8"
memory: "32000"
This configuration worked for vsphere.